addons/mail/static/src/core/common/suggestion_hook.js

```diff
@@ -20,9 +20,11 @@ class UseSuggestion {
                     ) {
                         return; // ignore obsolete call
                     }
+                    this.state.isFetching = true;
                     await this.suggestionService.fetchSuggestions(this.search, {
                         thread: this.thread,
                     });
+                    this.state.isFetching = false;
```

Comment:
It would be better to use try/finally to ensure we exit the fetching state if an error occurs.